lib/RexCmd.php

```diff
@@ -73,6 +76,22 @@ public static function phpExecutable(): string
                 if (is_executable($mampPhp.'php')) {
                     $path .= ':'.$mampPhp;
                 }
+
+                $homeDir = getenv('HOME');
+                if (!$homeDir) {
+                    $userInfo = posix_getpwuid(posix_getuid());
+                    if ($userInfo !== false) {
+                        $homeDir = $userInfo['dir'];
+                    }
+                }
+
+                if ($homeDir) {
+                    $herdPhp = $homeDir . '/Library/Application Support/Herd/bin/';
```

Comment:
ist der `homeDir` notwendig, oder würde es auch funktionieren mit

```suggestion
                $homeDir = getenv('HOME');
                if (!$homeDir) {
                    $userInfo = posix_getpwuid(posix_getuid());
                    if ($userInfo !== false) {
                        $homeDir = $userInfo['dir'];
                    }
                }

                if ($homeDir) {
                    $herdPhp = '/Library/Application Support/Herd/bin/';
```

analog mamp ?